gradlew

```diff
@@ -34,8 +34,8 @@
 #         * functions;
 #         * expansions «$var», «${var}», «${var:-default}», «${var+SET}»,
 #           «${var#prefix}», «${var%suffix}», and «$( cmd )»;
-#         * compound commands having a testable exit status, especially «case»;
-#         * various built-in commands including «command», «set», and «ulimit».
+#         * compound training.commands having a testable exit status, especially «case»;
```

Comment:
what

Reply:
it's gradle, I guess it's reflector or smth